### DISCUSSION TITLE
记录maa丢人时刻

### DISCUSSION THREAD

**Discussion by Yumi0606** [MaaAssistantArknights/MaaAssistantArknights#5963] [General]
笨笨maa 识别不到顶上的技能图标
![Snipaste_2023-08-16_10-07-15](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/136778556/b6391114-d1cd-4339-a02f-5745e2abd14e)

**Comment by SherkeyXD** [comment]
已知 issue #4962
